microovn/api/types/services.go

```diff
@@ -88,3 +88,29 @@ func NewRegenerateEnvResponse() RegenerateEnvResponse {
 		Errors:  make([]string, 0),
 	}
 }
+
```

Comment:
What's the reasoning for moving these vars/funcs here? I know they didn't feel super well in the `node.go` but I feel like they are very out of place in the package dedicated to API types. Did some package have problem with importing `node`?

Reply:
it says this in the message of the commit, it helps avoid circular dependencies, there are situations where we don't want to import node and all its dependencies just to use some types

Reply:
Thinking about this further, the service name definitions and list of valid services indeed make sense in this package, as they are related to making valid API requests/responses. I'm still not very convinced about the `CheckValidService` being in the package reserved for API type definitions, can we keep that in the `node` package?

> it says this in the message of the commit, it helps avoid circular dependencies

It does say that. However looking this commit I don't see any circular dependencies. It seems that any packages that now imports `types` (for these newly added vars/funcs) either already imports `node` or easily could. I was interested in whether I missed something.

What do you think about moving `SrvName`, `SrvChassis`, `SrvCentral`, `SrvSwitch` to the `types` package (as proposed) and keeping `CheckValidService` in the `node`? It seems that the only place that uses `SrvChassis` right now, will have to keep importing `node` anyway.

Reply:
nvm, let's just keep it all here. the `node` package indeed grew a lot in terms of dependencies with the addition of `Join`/`Leave` functions.